solidity/dashboard/src/components/Icons.jsx

```diff
@@ -518,4 +434,5 @@ export {
   Question,
   Home,
   FeesVector,
+  KeepDashboardLogo,
```

Comment:
Random question for my own edification: what do we gain by including these SVGs directly vs using an `img` or `picture` element to reference them? Just reducing the number of requests we're making, or is there another gain?

Reply:
We can style svg elements via css. In my opinion, is easier than changing svg styles inside img tag